This fixes a FTBFS where the C++ compiler dies on trying to use -std=gnu99
which is a C compiler option.

For context, '-xc' forces the compiler to interpret the code as C, which
means using clang, and not clang++.

upstream: No.

diff --git a/supportlibs/libsam3/Makefile b/supportlibs/libsam3/Makefile
index 8790dcb..dbc60c1 100644
--- a/supportlibs/libsam3/Makefile
+++ b/supportlibs/libsam3/Makefile
@@ -1,4 +1,4 @@
-CFLAGS := -Wall -g -O2 -std=gnu99
+CFLAGS := -Wall -g -O2 -std=gnu99 -xc
 
 SRCS := \
 	src/libsam3/libsam3.c \
